functions.php

```diff
@@ -334,6 +334,26 @@ function nightingale_register_required_plugins() {
 	tgmpa( $plugins, $config );
 }
 
+function get_theme_colours(){
```

Comment:
can this function please be moved to inc/custom-colours.php and renamed to start with nightingale_ so there is no risk of clash with core functionality.